.vscode/launch.json

```diff
@@ -1,4 +1,4 @@
-// A launch configuration that launches the extension inside a new window
```

Comment:
I believe so.  Because it's nice to have a run button for debugging and testing the extension inside vscode and this config provides that.